text/trace/0226-sampling-random-traceids.md

```diff
@@ -0,0 +1,214 @@
+# Non-power-of-two Probability Sampling using 56 random TraceID bits
+
+## Motivation
+
+The existing, experimental [specification for probability sampling using TraceState](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/tracestate-probability-sampling.md)
+supporting Span-to-Metrics pipelines is limited to powers-of-two
+probabilities and is designed to work without making assumptions about 
+TraceID randomness.
+
+The term "Tail sampling" is in common use to describe _various_ forms
+of sampling that take place after a span starts.  The term "Tail" in
+this phrase distinguishes other techniques from head sampling, however
+the term is only broadly descriptive.
+
+Head sampling requires the use of TraceState to propagate context
+about sampling decisions parent spans to child spans.  With sampling
+information included in the TraceState, spans can be labeled with their
+effective adjusted count, making it possible to count spans as they
+arrive at their destination in real time, meaning before assembling
+complete traces.
+
+Here, the term Intermediate Span Sampling is used to describe sampling
+performed on individual spans at any point in their collection path.
+Like Head sampling, Intermediate Span Sampling benefits from being
+consistent, because it makes recovery of complete traces possible
+after spans have independently sampled.  On the other hand, when "Tail
+sampling" refers to sampling of complete traces, sampling consistency
+is not an important property.
```

Comment:
I think the point is that when "tail sampling" means "making the decision when you have the entire trace available", you're by definition making a self-consistent decision since there's only one instance making the decision. You either get all the spans or none of them. It doesn't really matter very much in this context whether a different instance would have come to the same decision.

It's also the case that we are sometimes talking about non-probabalistic decisions like "is there an exception span event in this trace?"